app/browser/api/topSites.js

```diff
@@ -85,6 +89,7 @@ const getTopSiteData = () => {
   let sites = historyState.getSites(state)
     .filter((site, key) => !isSourceAboutUrl(site.get('location')) &&
       !isPinned(state, key) &&
+      !isIgnored(state, site.get('key')) &&
```

Comment:
can we use just `key` here same as for `isPinned`?

Reply:
ok done